widgetssdk/src/main/java/com/glia/widgets/core/engagement/domain/GliaOnEngagementEndUseCase.java

```diff
@@ -45,6 +49,12 @@ public void run() {
             fileAttachmentRepository.detachAllFiles();
             removeScreenSharingNotificationUseCase.execute();
             removeCallNotificationUseCase.execute();
+
+            engagement.getSurvey((survey, exception) -> {
+                if (survey != null) {
+                    Log.d("Survey", survey.toString());
+                }
+            });
```

Comment:
Do we need these changes on the version update?

Reply:
I'd remove this - we have doc available for that :)

Reply:
Removed